### PR TITLE
Add checkout fees, update min/max amounts, ..

### PR DESCRIPTION
**Pull request by liayoo**
- Added checkout_fee_rate (0.1%)
- Increased min & max checkout amounts
- Changed eth token bridge configs for testnet-* from eth mainnet to ropsten
- Added a blockchain_params.json placeholder for the mainnet

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM tho. What about double checking with @platfowner ?

File: blockchain-configs/afan-shard/blockchain_params.json [hunk 1]
```diff
@@ -9,9 +9,10 @@
           "0": {
             "network_id": 1,
             "token_pool": "0xBBB2219cD5eACc54Ce95deF7a67dDe71C8241891",
-            "min_checkout_per_request": 1000,
-            "max_checkout_per_request": 10000,
+            "min_checkout_per_request": 10000,
+            "max_checkout_per_request": 100000,
```
Comment (platfowner) [inline blockchain-configs/afan-shard/blockchain_params.json:13]: question: is this correct that we don't need per-day max as a user cannot create multiple pending requests?

File: db/functions.js [hunk 2]
```diff
@@ -1193,8 +1194,9 @@ class Functions {
     if (amountValidated !== true) {
       return this.returnFuncResult(context, amountValidated);
     }
+    const fee = amount * checkoutFeeRate;
```
Comment (platfowner) [inline db/functions.js:1197]: minor: what about defining e.g. 'transferAmount' like 'refundAmount' here or uring 'fee' in _closeCheckout as well?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments and questions.
Thanks!

File: blockchain-configs/afan-shard/blockchain_params.json (hunk 1 above)
Comment (liayoo) [reply blockchain-configs/afan-shard/blockchain_params.json:13]: It's actually used for check-outs. Check-ins do not allow a user to create multiple pending requests with the same <network name, chain id, token id, sender address>, but for check-outs, there are no such limits.

Comment (platfowner) [reply blockchain-configs/afan-shard/blockchain_params.json:13]: Got it, thanks!

File: db/functions.js (hunk 2 above)
Comment (liayoo) [reply db/functions.js]: Defined transferAmount to match refundAmount. Thanks!

**Comment by liayoo** [comment]
Thanks for the review! Now merging~

**Pull request merged**